src/Maker/MakeEntity.php

```diff
@@ -160,6 +163,22 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             'Entity\\'
         );
 
+        $mappingDriver = $this->doctrineHelper->getMappingDriverForNamespace($entityClassDetails->getFullName());
+
+        if ($mappingDriver instanceof AttributeDriver) {
+            if (\PHP_VERSION_ID < 80000) {
+                throw new RuntimeCommandException(sprintf('Your PHP version (%s) does not supports attributes, PHP 8 is needed.', \PHP_VERSION));
+            }
+
+            if (!$this->doctrineHelper->isDoctrineSupportingAttributes()) {
+                throw new RuntimeCommandException('Your doctrine version does not supports attributes. You need at least doctrine in version 2.9');
+            }
+
+            if (Kernel::VERSION_ID < 50200) {
+                throw new RuntimeCommandException('Your symfony version does not support attributes. Attribute supports begins with >= Symfony 5.2 ');
```

Comment:
What feature in `symfony/http-kernel: 5.2` do we need?

Beware, symfony is a set of component. People can install `http-kernel 5.1` alongside `dependency-injection 5.2`